src/GUI/RevEng.Core.60/Functions/SqlServerFunctionScaffolder.cs

```diff
@@ -47,6 +47,12 @@ protected override string WriteDbContext(ModuleScaffolderOptions scaffolderOptio
             Sb.AppendLine("using System.Data;");
             Sb.AppendLine("using System.Linq;");
             Sb.AppendLine($"using {scaffolderOptions.ModelNamespace};");
+
+            if (scaffolderOptions.UseSchemaNamespaces)
+            {
+                model.Routines.Select(r => r.Schema).Distinct().ToList().ForEach(schema => Sb.AppendLine($"using {scaffolderOptions.ModelNamespace}.{schema}"));
```

Comment:
This only selects the schemas from stored procedures and functions, this seem wrong